deeppavlov/configs/elmo/elmo-1b-benchmark.json

```diff
@@ -0,0 +1,86 @@
+{
+  "dataset_reader": {
+    "class_name": "file_paths_reader",
+    "data_path": "{DOWNLOADS_PATH}/elmo-1b-benchmark/data/1-billion-word-language-modeling-benchmark-r13output/",
+    "train": "training-monolingual.tokenized.shuffled/*",
+    "valid": "heldout-monolingual.tokenized.shuffled/*"
+  },
+  "dataset_iterator": {
+    "class_name": "elmo_file_paths_iterator",
+    "seed": 31415,
+    "unroll_steps": 20,
+    "max_word_length": 50,
+    "n_gpus": 3,
+    "shuffle": false,
+    "bos":"<S>",
+    "eos":"</S>",
+    "save_path": "{MODELS_PATH}/elmo-1b-benchmark/vocab-2016-09-10.txt",
+    "load_path":"{MODELS_PATH}/elmo-1b-benchmark/vocab-2016-09-10.txt"
+  },
+  "chainer": {
+    "in": [
+      "x_char_ids"
+    ],
+    "in_y": [
+      "y_token_ids"
+    ],
+    "pipe": [
+      {
+        "class_name": "elmo_model",
+        "options_json_path": "{MODELS_PATH}/elmo-1b-benchmark/options.json",
+        "unroll_steps": 20,
+        "batch_size": 128,
+        "save_path": "{MODELS_PATH}/elmo-1b-benchmark/ckpts/current_ckpt/model",
+        "load_path": "{MODELS_PATH}/elmo-1b-benchmark/ckpts/current_ckpt/model",
+        "in": ["x_char_ids", "y_token_ids"],
+        "in_y": [],
+        "n_gpus": 3,
+        "out": ["loss"]
+      }
+    ],
+   "out": [
+    "x_char_ids",
+    "y_token_ids"
+  ]
+  },
+  "train": {
+    "epochs": 20,
+    "batch_size": 128,
+    "log_every_n_batches": 100,
+    "val_every_n_epochs": 1,
+    "validation_patience": 4,
+    "metric_optimization": "minimize",
+    "metrics": [
+      {
+        "name": "elmo_loss2ppl",
+        "inputs": ["loss"]
+      }
+    ],
+    "tensorboard_log_dir": "{MODELS_PATH}/elmo-1b-benchmark/logs"
+  },
+  "metadata": {
+    "variables": {
+      "ROOT_PATH": "~/.deeppavlov",
+      "DOWNLOADS_PATH": "{ROOT_PATH}/downloads",
+      "MODELS_PATH": "{ROOT_PATH}/models"
+    },
+    "requirements": [
+      "{DEEPPAVLOV_PATH}/requirements/tf.txt",
+      "{DEEPPAVLOV_PATH}/requirements/tf-hub.txt"
+    ],
+    "download": [
+      {
+        "url": "http://files.deeppavlov.ai/deeppavlov_data/1-billion-word-language-modeling-benchmark-r13output.tar.gz",
+        "subdir": "{DOWNLOADS_PATH}/elmo-1b-benchmark/data"
+      },
+      {
+        "url": "https://s3-us-west-2.amazonaws.com/allennlp/models/elmo/vocab-2016-09-10.txt",
```

Comment:
If files are moved from allennlp, these downloads will not work. There is a point in copying required files to files.deeppavlov.ai